components/waitwhile/actions/remove-customer/remove-customer.ts

```diff
@@ -0,0 +1,33 @@
+import waitwhile from "../../app/waitwhile.app";
+import { defineAction } from "@pipedream/types";
+
+export default defineAction({
+  name: "Remove Customer",
+  version: "0.0.1",
+  key: "waitwhile-remove-customer",
+  description: "Remove a customer",
+  props: {
+    waitwhile,
+    customerId: {
+      propDefinition: [
+        waitwhile,
+        "customerId",
+      ],
+    },
+  },
+  type: "action",
+  methods: {},
+  async run({ $ }) {
+
+    try {
+      const data = await this.waitwhile.removeCustomer(this.customerId);
+      $.export("summary", "Successfully removed a customer");
+      return data;
+    } catch(error){
+        const statusCode = error[Object.getOwnPropertySymbols(error)[1]].status;
```

Comment:
Looks like an indentation error (you should use a auto format tool to avoid this kind of problem)